### DISCUSSION TITLE
$XCH nose diving???

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#6687] [Q&A]
What is going on? It is <$440 now!
What is the incentive of continued plotting? i.e. hold the bag and wish for the best to happen?

**Comment by room101-dev** [comment]
> What is going on? It is <$440 now!
> What is the incentive of continued plotting? i.e. hold the bag and wish for the best to happen?

Essentially they cracked a new noob sucker stream with 'chia', its green, gay, & clean; What's a mother not to like? There are still 100's of influencer YT videos in all human languages on YT all reading from the same paid script of 'caleb' all saying 'Get rich plotting chia on your old HDD"; This thing has legs, but the price, oh well that will go down & hard. It was always just about the IPO, and the space (EB) will sell the IPO, the price of XCH doesn't matter to the chia-net dev ppl in Calif, because they don't have any 'chia'(xch) their end in the deal was always the IPO ( NYSE trade value $10Billion USD, next Red-Hat Crypto support fortune 500 company, to be sold to IBM, 3+ years future )

It's going down, IMHO around $50, maybe lower. Here's the deal, it was a chinese IOU back first six months of this year, long before the main-net of 05may2021 it was trading as an IOU on chinese exchanges, just like its brother bitcoin-gold was an IOU some 5 years ago, long before main-net, all the same people rinse&repeat. Only difference this time is that the chinese billionaires are using a faux dev-team in Calif(SV aka chia-net-inc ) to act as the base, with bitcoin-gold they used a team in UKRAINE, but hid the chinese ownership.

IMHO the entire cat is out of the bag, and yes the only bag holders for XCH will be the west. This thing was setup and pumped, and the HDD companys front runned, the chia-net team will get their IPO, as a sack of feces can be sold to the public, and lastly bitcoin-gold is still around, so might xch in 5+ years of today.
Funny of all is chia-explorer is still saying the price is $550, but coinmarketcap is saying $440, so the real price is probably already broken $100, my guess is the IOU folks in china will liquidate this week.

...

Should you mine? Well of course not, think about this,what exactly will these useless plots be used for in the future? We already know they can be faked, some 20% of plots submitted to HPOOL are fake, which means the entire plot-create has to be redesigned, and of course the portable-plot protocol is just fantasy coming out of team BS in Calif. Remember all they get is the IPO, and they can't lose, as crypto is hot, and this shit-coin is 99% marketing, and the EB  space is still growing, now at +22EB, at 4%/day. The momentum for the morons is constant now, and the price sure will fall, but reality was disconnected long ago.

So sure plot on, but just remember its just random numbers on a disk, that burns up SSD drives and HDD drives. It all comes down to how much HD you want to kill with no ROI.

**Reply by 69chargr** [reply]
Wow. I mean. WOW! Lol

**Reply by jeblumena** [reply]
Lols... this guy has a colorful mind spewing feces everywhere...

**Reply by pluto2021** [reply]
Although negative, but good information from a neutral stand point of view. The behind scene deals are always shielded from the public. We need to thank room101-dev for the information (taking the likely facts out of opinions), with a grain of salt of course. The public is always getting the leftovers (feces :D) of major deals, as usual. IPOs usually happen with 95% of the gains already taken out of the table before the public. Take a look at the SPAC ordeals happened recently, not only did many retail investors not gain anything, it was a blood bath..., yet nothing lost on the SPAC companies side.

Not sure how we, who have contributed to the significant netspace growth, could benefit from the Chia IPO???  Any suggestions? say are we considered any sort of "private" investors pre-IPO? How do we take a fair share of the IPO? If we only participate in the netspace growth and the netspace is used to justify the valuation of the company for an IPO, it is unfair that only the Chia company gets the pre-IPO shares at dirty cheap price and make 95% of the money!  I would like to make a formal request to distribute the pre-IPO shares to each farmer according to their contributions! We cannot support this Chia network if we are not part of this IPO deal!!!

**Reply by pluto2021** [reply]
I would guess that the current netspace growth might primarily come from two groups of people: the innocent solos who do not know the behind scene deals, and those who would benefit from this IPO as the more netspace it is, the higher the valuation would be. So we are doing the hard work, yet the Chia investors (excluding the solo farmers) would be counting the money???

How about distributing the pre-IPO shares to each farmer in some mechanism based on the number of plots made and validated? I would suggest forgetting the pools for now. It is no longer economical. It is now all about the IPO and maximizing the valuation.

**Reply by Lenkus1** [reply]
'with a grain of salt ' How bout a truck full?

**Reply by pluto2021** [reply]
It is always wise to take useful information out of even a lot more that are not useful, and focus on the useful part only.

The lower the $XCH, the lower the valuation would be. So it is the interest of the sponsors and the Chia company to drive up the price. They would probably use the cash reserve to buy at the low and take double profits when $XCH get higher. So it might not be a good idea to flood the market with the pre-mined XCH yet. 

At current net space, with a linear growth rate of 700PiB/day, and a price of $421 (at the time of writing), each plotted TiB would only be able to bring 0.03XCH or $12.56 of revenue in a year, not including electricity cost. This is far less than the per TiB cost of new HDD consumer pricing + plotting rig + plotting alone for now. Risky investment that can only win if the $XCH goes back up significantly higher, or you are already part of the IPO deal.

**Reply by room101-dev** [reply]
> Although negative, but good information from a neutral stand point of view. The behind scene deals are always shielded from the public. We need to thank room101-dev for the information (taking the likely facts out of opinions), with a grain of salt of course. The public is always getting the leftovers (feces :D) of major deals, as usual. IPOs usually happen with 95% of the gains already taken out of the table before the public. Take a look at the SPAC ordeals happened recently, not only did many retail investors not gain anything, it was a blood bath..., yet nothing lost on the SPAC companies side.
> 
> Not sure how we, who have contributed to the significant netspace growth, could benefit from the Chia IPO??? Any suggestions? say are we considered any sort of "private" investors pre-IPO? How do we take a fair share of the IPO? If we only participate in the netspace growth and the netspace is used to justify the valuation of the company for an IPO, it is unfair that only the Chia company gets the pre-IPO shares at dirty cheap price and make 95% of the money! I would like to make a formal request to distribute the pre-IPO shares to each farmer according to their contributions! We cannot support this Chia network if we are not part of this IPO deal!!!

Dream on, big guy, the world ain't fair. The deal was made last year, and everything is turning fold according to plan.

Bram even said "Why do suckers, who followed me feel entitled"?, good question indeed, if someone tells u they're going to screw u, and u follow them to hell, why would u expect a tip? The general problem here is that the morons get nuttin, but they don't get it, cuz they're morons.

The HDD company's financed the HDD-CHIA to the moon scam, they profit by 10x increase in HDD price. Ppl who bit the bullet or bait, can't really even be called 'entitled', everybody on earth would call them stupid. Bram was being gentle when he called U all 'entitled', yep your entitled to zero.

p.s. dude, thanks for the support, but this ain't rocket science, these people have been rinse&repeating this crap for as long as crypto has been around the only thing new here is that CHIA targeted, Young, green, gay, and stupid. The people in China, who own all this crap must be laughing their arses off, but it dont' matter, cuz they'll do it again, because the noobs have a memory retention of days not months.

**Reply by pluto2021** [reply]
No wonder Bram's smile does not look honest. This is sad. Bram took the best use of the human greed. We all become his modern slaves.

**Reply by 69chargr** [reply]
You must really hate Vitalik then! 🤣🤣🤣

**Reply by pluto2021** [reply]
Well, for ICO average joe may be able to stake to participate to get the most gain. For this IPO thing, it is completely behind scene deal along with the wall street mafia. If you are part of the deal, I would have to admire you.

**Reply by 69chargr** [reply]
So a 72million ether premine doesn't bother you?! 🤣🤣🤣

**Reply by pluto2021** [reply]
Things look norm do not necessarily mean they are fair. If you do not fight for your share, you will not get it by default. Helping others getting rich using my own resource without giving me the share I deserve makes me feel like,

![jackass-3](https://user-images.githubusercontent.com/82697187/121793960-e9b05300-cbb8-11eb-9183-4ca5833f855b.jpg)

**Reply by 69chargr** [reply]
They don't owe you anything. Nothing. Your "fair share" is your reward based on your percent of total netspace. Just like your ETH reward is based on your percent of total network hashrate. That's as fair as can possibly be expected. Anything more is "fair" to you and "unfair" to everyone else.

**Reply by pluto2021** [reply]
Well, this Chia is the first Coin to go IPO, not ICO. This makes things very different. The fair market value of the IPO depends on the netspace plotted. Getting a fair share of the netspace includes both the humble rewards that could not cover the expenditures, but also should include its fair share of the pre-IPO shares that would be distributed. I would assume that the pre-IPO shares have been or would be distributed among the major investors and the Chia company, and has nothing to do with all farmers who have contributed to the netspace, the very thing the IPO valuation is depending upon! If a farmer contributes to the netspace that is used to estimate the valuation of the Chia company, why he is not counted for the pre-IPO share distribution??? I may take down all my plots off line before a scheduled IPO by the time my plots would not be able to earn me much anyway!!! Solo farmers should go for a netspace strike before the IPO to show that solo farmers are the very thing Chia depends and deserves the shares.

**Reply by 69chargr** [reply]
Where are you getting the idea that company valuation = netspace? The promises of Chialisp and the attractiveness of low power consumption relative to PoW chains are what drives valuation.

**Reply by pluto2021** [reply]
The netspace is not all but certainly a significant part of that equation. I said "depends", not "=". Need to be accurate. Would Chia worth anything if there is little netspace? besides all Chialsp and the "attractiveness"? BTW, how would you calculate the valuation of the "attractiveness" that is so subjective? It is certainly attractive to whoever is on the IPO deal behind the scene. For solo farmers, any real incentives beyond an empty wish for the $XCH to skyrocket some day? These rewards are really leftovers, classic for IPO. Upon IPO, the Chia team and the private investors (excluding farmers who paid for the netspace) would get rich regardless.  For traditional IPO, private investors would need to pay for the netspace!!! Now it appears that Bram used this Chia crypto as a bait for others (farmers) to pay for the netspace he needs to go IPO!!! I believe that this needs to be reviewed by the SEC, seriously.

**Reply by 69chargr** [reply]
Hahahah!! So by your definition Ethereum is a scam too. 72 million ETH premine right?

**Reply by pluto2021** [reply]
I do not want to compare ETH first. So please try not to lead to that direction. Whether this Chia scheme is fair or not has nothing to do with ETH. My comments on Chia also do not have anything to do with ETH.  Secondly, I never said Chia is a scam. You said that. So don't make me wrong.  It can be a good project if farmers can get their real fair share, and profitable incentives that are sustainable.

**Reply by 69chargr** [reply]
You don't want to compare because it's literally the exact same scenario! No matter what coin you're talking about, your proportion of the reward is based solely on your share of the total work. The ONLY difference at the moment is that there is no pooling protocol in place. Go solo mine ETH or BTC and see what you get because that's what we're all doing with XCH untill actual pooling protocol is released. This is literally no different than any other coin. Stop spreading FUD and make an effort to educate yourself and at least try to be intellectually honest about the facts.
